tests/providers/aws/services/ec2/ec2_instance_multiple_eni/ec2_instance_multiple_eni_test.py

```diff
@@ -0,0 +1,317 @@
+        ), mock.patch(
+            "prowler.providers.aws.services.ec2.ec2_instance_multiple_eni.ec2_instance_multiple_eni.ec2_client",
+            new=EC2(aws_provider),
+        ):
+            from prowler.providers.aws.services.ec2.ec2_instance_multiple_eni.ec2_instance_multiple_eni import (
+                ec2_instance_multiple_eni,
+            )
+
+            check = ec2_instance_multiple_eni()
+            result = check.execute()
+
+            assert len(result) == 0
+
+    @mock.patch("botocore.client.BaseClient._make_api_call", new=mock_make_api_call_v2)
+    def test_ec2_instance_no_eni(self):
+
+        from prowler.providers.aws.services.ec2.ec2_service import EC2
+
+        aws_provider = set_mocked_aws_provider([AWS_REGION_US_EAST_1])
+
+        with mock.patch(
+            "prowler.providers.common.provider.Provider.get_global_provider",
+            return_value=aws_provider,
+        ), mock.patch(
+            "prowler.providers.aws.services.ec2.ec2_instance_multiple_eni.ec2_instance_multiple_eni.ec2_client",
+            new=EC2(aws_provider),
+        ):
+            from prowler.providers.aws.services.ec2.ec2_instance_multiple_eni.ec2_instance_multiple_eni import (
+                ec2_instance_multiple_eni,
+            )
+
+            check = ec2_instance_multiple_eni()
+            result = check.execute()
+
+            assert len(result) == 1
+            assert result[0].status == "PASS"
+            assert result[0].region == AWS_REGION_US_EAST_1
+            assert result[0].resource_id == "i-0123456789abcdef0"
+            assert (
+                result[0].status_extended
+                == "EC2 Instance i-0123456789abcdef0 has no network interfaces attached."
+            )
+
+    @mock.patch("botocore.client.BaseClient._make_api_call", new=mock_make_api_call)
+    def test_instance_multiple_enis(self):
+
+        from prowler.providers.aws.services.ec2.ec2_service import EC2
+
+        aws_provider = set_mocked_aws_provider([AWS_REGION_US_EAST_1])
+
+        with mock.patch(
+            "prowler.providers.common.provider.Provider.get_global_provider",
+            return_value=aws_provider,
+        ), mock.patch(
+            "prowler.providers.aws.services.ec2.ec2_instance_multiple_eni.ec2_instance_multiple_eni.ec2_client",
+            new=EC2(aws_provider),
+        ):
+            from prowler.providers.aws.services.ec2.ec2_instance_multiple_eni.ec2_instance_multiple_eni import (
+                ec2_instance_multiple_eni,
+            )
+
+            check = ec2_instance_multiple_eni()
+            result = check.execute()
+
+            assert len(result) == 1
+            assert result[0].status == "FAIL"
+            assert result[0].region == AWS_REGION_US_EAST_1
+            assert result[0].resource_id == "i-0123456789abcdef0"
+            assert (
+                result[0].status_extended
+                == "EC2 Instance i-0123456789abcdef0 uses multiple ENIs: "
```

Comment:
Can you prettify this message please?

Reply:
What do you mean by prettify? Should I add global attributes for the instance id and enis ids?

Reply:
Something like: 
```
"EC2 Instance i-0123456789abcdef0 uses only one ENI: (EFAs: ['eni-2'], Trunks: ['eni-1', 'eni-3'].)"
```